.github/workflows/ios-sandbox-deployment-dev.yml

```diff
@@ -0,0 +1,123 @@
+        uses: actions/checkout@v3
+
+      - name: Set up ruby env
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7.6
+          bundler-cache: true
+
+      - name: Restore cached Pods
+        uses: actions/cache/restore@v3
+        with:
+          path: Pods
+          key: ${{ runner.os }}-pods-${{ hashFiles('**/Podfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-pods-
+
+      - name: CocoaPod Install
+        run: pod install
+
+      - name: Save Updated Pods
+        id: cache-primes-save
+        uses: actions/cache/save@v3
+        with:
+          path: Pods
+          key: ${{ runner.os }}-pods-${{ hashFiles('**/Podfile.lock') }}
+
+      - name: Get last version from tags
+        run: |
+          git fetch --prune --unshallow
+          echo "VERSION=$(git describe --tags)" >> $GITHUB_ENV
+
+      - name: Run iOS Unit Testing for Dev SDK
+        run: bundle exec fastlane test
+
+      - name: Run iOS Sandbox Dev Deployment
+        run: |
+          curl \
+             -X POST \
+             -H "Accept: application/vnd.github.v3+json" \
+             -H "Authorization: token ${{ secrets.GPR_API_KEY }}" \
+             https://api.github.com/repos/Neuro-ID/neuroid-ios-sdk-sandbox/dispatches \
+             -d '{"event_type":"on-demand-testflight-dev","client_payload":{"version":"5.ios-${{env.VERSION}}", "message": "${{ github.event.head_commit.message }}"}}'
+
+      - name: Run iOS SwiftUI Sandbox Dev Deployment
+        run: |
+          curl \
+            -X POST \
+            -H "Accept: application/vnd.github.v3+json" \
+            -H "Authorization: token ${{ secrets.GPR_API_KEY }}" \
+            https://api.github.com/repos/Neuro-ID/neuroid-ios-sdk-sandbox-swiftui/dispatches \
+            -d '{"event_type":"on-demand-testflight-dev","client_payload":{"version":"5.ios-${{env.VERSION}}", "message": "${{ github.event.head_commit.message }}"}}'
+
+      - name: Run iOS Storyboard Sandbox Dev Deployment
+        run: |
+          curl \
+            -X POST \
+            -H "Accept: application/vnd.github.v3+json" \
+            -H "Authorization: token ${{ secrets.GPR_API_KEY }}" \
+            https://api.github.com/repos/Neuro-ID/neuroid-ios-sdk-sandbox-storyboard/dispatches \
+            -d '{"event_type":"on-demand-testflight-dev","client_payload":{"version":"5.ios-${{env.VERSION}}", "message": "${{ github.event.head_commit.message }}"}}'
+
+      - name: Run ReactNative Sandbox Dev Deployment
+        run: |
+          curl \
+              -X POST \
+              -H "Accept: application/vnd.github.v3+json" \
+              -H "Authorization: token ${{ secrets.GPR_API_KEY }}" \
+              https://api.github.com/repos/Neuro-ID/neuroid-reactnative-sdk/dispatches \
+              -d '{"event_type":"on-demand-testflight-dev","client_payload":{"version":"5.ios-${{env.VERSION}}", "message": "${{ github.event.commits[0].message }}"}}'
```

Comment:
Just FYI - message reference looks different than the rest (although they probably evaluate to the same value).